csi/moac/test/volumes_test.js

```diff
@@ -336,7 +371,7 @@ module.exports = function () {
         state: 'NEXUS_ONLINE',
         children: [
           {
-            uri: `nvmf:///blabla/${UUID}`,
+            uri: `nvmf://blabla/${UUID}`,
```

Comment:
i think the original /// were correct

Reply:
I always confuse these. The general scheme is `protocol://host:port/path`, right? So if there is no host part then you get 3 slashes in row. With nvmf there is the host part or isn't? It's not really important as long as we use the same string here and in the assert that is testing the value, but would be nice to get it right.

Reply:
Right, with nvmf you need the host part I believe. Ah is blabla the host path?

Reply:
yea, very inaccurate approximation of what you would normally find there :-) But there are places where I got it even worse without host part just nvmf://${UUID} ... :-)

Reply:
hehe then it's now fixed :-)